kubernetes_tests/test_kubernetes_executor.py

```diff
@@ -266,3 +266,65 @@ def test_integration_run_dag_with_scheduler_failure(self):
         )
 
         assert self._num_pods_in_namespace('test-namespace') == 0, "failed to delete pods in other namespace"
+
+    def test_integration_run_dag_with_running_task_pod_kill(self):
```

Comment:
Can you add comments / docstrings to say what this test is actually testing, please

Reply:
Done